bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/BulkMutation.java

```diff
@@ -385,8 +385,11 @@ public synchronized void run() {
           retryId = Long.valueOf(
             asyncExecutor.getOperationAccountant().registerComplexOperation(createRetryHandler()));
         }
+        long start = clock.nanoTime();
         mutateRowsFuture = asyncExecutor.mutateRowsAsync(currentRequestManager.build());
-        currentRequestManager.lastRpcSentTimeNanos = clock.nanoTime();
+        long now = clock.nanoTime();
+        BulkMutationsStats.getInstance().markThrottling(now - start);
```

Comment:
This looks a bit weird. Are you intending on mutateRowsAsync blocking to achieve throttling?